src/main/resources/view/TravelPlannerPanel.fxml

```diff
@@ -8,7 +8,7 @@
 <?import javafx.scene.control.ListCell?>
 <VBox fx:id="travelPlanPanel" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
     <padding>
-        <Insets top="5" right="5" bottom="5" left="5" />
+        <Insets top="5" right="5" bottom="5" left="5"  />
```

Comment:
Here also. Or is it deleted whitespace haha i not sure

Reply:
Ok i can delete that one whitespace